orcid-web/src/main/java/org/orcid/frontend/web/controllers/ManageProfileController.java

```diff
@@ -957,7 +957,7 @@ public ModelAndView authorizeDelegatesRequest(@RequestParam("key") String key) {
         }
     }
     
-    @RequestMapping(value = { "/get-trusted-orgs" }, method = RequestMethod.GET)
+    @RequestMapping(value = { "/get-trusted-orgs.json" }, method = RequestMethod.GET)
```

Comment:
We will need to also merge: https://github.com/ORCID/orcid-angular/pull/1602

Since I already did the change on the UI on the previous release. (Sorry did not notice there was a card about it assigned to you Camelia).